bom-latest/pom.xml

```diff
@@ -78,6 +79,11 @@
                 <artifactId>workflow-job</artifactId>
                 <version>2.39</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins.workflow</groupId>
+                <artifactId>workflow-multibranch</artifactId>
+                <version>2.16</version>
```

Comment:
This is `no-test-jar=false` so you should define a POM property and manage the `tests`-classifier artifact as well.